### ISSUE TITLE
Jupyter Notebook - Blank Screen - after pip install jupyter

### ISSUE THREAD

**Issue by oesalgado** [jupyter/notebook#4469]
I'm trying to get jupyter notebook to simply work when install it on a new test environment using Anaconda prompt.

These are the steps I'm using:

- Open Anaconda Prompt
- Create a new environment using: `conda create -n $nameofmyenvironment pip python=3.5`
- Activate environment: `activate tflowtest` (name of my environment)
- Open Jupyter notebook command: `jupyter notebook`   |   https://prnt.sc/mw1r58
-This actually ends up opening notebook with no issues  |   https://prnt.sc/mw1pi9

Now this is when it stops working properly:

- Use the jupyter installation command: `pip install jupyter`
-After the installation goes through with no issues this is what I get: https://prnt.sc/mw1t9y

My console returns no errors: https://prnt.sc/mw1tw3

I've uninstalled and reinstalled the latest version of Anaconda and still getting the same result.

Any suggestions and feedback is truly appreciated.

**Comment by ciaranlangton** [comment]
I've just submitted a PR to fix this issue, see #4467, #4468 

TL;DR the MIME Content-Type returned for the JavaScript files is `text/plain` rather than `application/javascript` which the browser then refuses to load.

**Comment by ciupicri** [comment]
This seems to be a duplicate of issue #4469.

**Comment by minrk** [comment]
This will be fixed in 5.7.7
